components/autocomplete/index.js

```diff
@@ -243,6 +243,16 @@ export class Autocomplete extends Component {
 			range.setStartAfter( child );
 		}
 		range.deleteContents();
+
+		let inputEvent;
+		if ( undefined !== window.InputEvent ) {
+			inputEvent = new window.InputEvent( 'input', { bubbles: true, cancelable: false } );
+		} else {
+			// IE11 doesn't provide an InputEvent constructor.
+			inputEvent = document.createEvent( 'UIEvent' );
```

Comment:
I was following @brandonpayton's lead on this... but looking at it, for consistency, shouldn't it be an InputEvent?

Reply:
Perhaps. :) Since this is an IE quirk, I don't know. I brought up `Event` because the typical examples out there on how to "deal with IE" use it instead of other event types.

Reply:
Actually, `InputEvent` is not supported at all, even with the fallback interface. It has to be either `Event` or `UIEvent`.

Reply:
I think we would be fine using `Event`, but the reason is that the `InputEvent` interface is based on `UIEvent` in the spec:
https://w3c.github.io/uievents/#idl-inputevent